sentry_sdk/integrations/django/asgi.py

```diff
@@ -91,3 +97,52 @@ async def sentry_wrapped_callback(request, *args, **kwargs):
             return await callback(request, *args, **kwargs)
 
     return sentry_wrapped_callback
+
+
+def _asgi_mixin_factory(_check_middleware_span):
+    # type: (F) -> Any
+    """
+    Class factory that generates a class that deals with handling requests
+    in async mode.
+    """
```

Comment:
Please clarify in both method name and docstring that this is a mixin class for middlewares specifically